lib/wallet/src/Cardano/Wallet/Write/Tx.hs

```diff
@@ -826,10 +896,22 @@ evaluateMinimumFee era pp tx kwc =
         Shelley.evaluateTransactionFee pp tx nKeyWits
 
     bootWitnessFee :: Coin
-    bootWitnessFee =
-        if nBootstrapWits > 0
-        then error "evaluateMinimumFee: bootstrap witnesses not yet supported"
-        else mempty
+    bootWitnessFee = Coin $
+        intCast $ feePerByte * bytes
+      where
+        bytes :: Natural
+        bytes = sizeOf_BootstrapWitnesses $ intCast nBootstrapWits
```

Comment:
Suggestion: be a little more explicit that is a _count_ of bytes, rather than the byte values themselves.

```suggestion
        intCast $ feePerByte * byteCount
      where
        byteCount :: Natural
        byteCount = sizeOf_BootstrapWitnesses $ intCast nBootstrapWits
```